recipe/meta.yaml

```diff
@@ -32,14 +36,20 @@ test:
   ] %}
   {% for lib in readline_libs %}
     - test -f ${PREFIX}/lib/{{ lib }}.a
-    - test -f ${PREFIX}/lib/{{ lib }}.so      # [linux]
-    - test -f ${PREFIX}/lib/{{ lib }}.dylib   # [osx]
+    - test -f ${PREFIX}/lib/{{ lib }}${SHLIB_EXT}  # [not win]
   {% endfor %}
 
 about:
   home: https://cnswww.cns.cwru.edu/php/chet/readline/rltop.html
-  license: GPL3
+  license: GPL-3.0
+  license_file: COPYING
   summary: library for editing command lines as they are typed in
+  description: |
+    The standard Python readline extension statically linked against the GNU
+    readline library.
+  doc_url: https://pypi.python.org/pypi/readline
```

Comment:
I'm confused by this. Why do we reference python-gnureadline?

Reply:
I copied-n-pasted without checking. It is wrong upstream! Thanks!